pkg/controller/containerjfr/resource_definitions/resource_definitions.go

```diff
@@ -287,10 +301,28 @@ func NewCoreContainer(cr *rhjmcv1beta1.ContainerJFR, specs *ServiceSpecs, tls *T
 			SubPath:   CAKey,
 			ReadOnly:  true,
 		}
-		// TODO add mechanism to add other certs to /truststore
 
 		mounts = append(mounts, keystoreMount, caCertMount)
 
+		secretMounts := []corev1.VolumeMount{}
+		for _, secret := range cr.Spec.TrustedCertSecrets {
+			var key string
+			if secret.CertificateKey != nil {
+				key = *secret.CertificateKey
+			} else {
+				key = rhjmcv1beta1.DefaultCertificateKey
+			}
+			mount := corev1.VolumeMount{
+				Name:      secret.SecretName,
+				MountPath: fmt.Sprintf("/truststore/%s-%s", secret.SecretName, key),
```

Comment:
I realized there's still potential for naming collision here. Say we have two secrets:
1. foo-bar, with key baz
2. foo, with key bar-baz
These are both created as foo-bar-baz.

Underscores aren't valid in Kubernetes object names. So I think we should be okay if we use an underscore instead of a hyphen as a separator between the secret name and key when mounting the file.

Reply:
Oh okok, yeah that would prevent that kind of collision!